tools/docker/docker-compose.yml

```diff
@@ -48,6 +48,8 @@ services:
       - .env
     environment:
       - HOST_PORT=${PORT_WORDPRESS:-80}
+      - WP_ADMIN_USER=${WP_ADMIN_USER:-jp_docker_acct}
+      - WP_ADMIN_PASSWORD=${WP_ADMIN_PASSWORD:-jp_docker_acct}
```

Comment:
Setting these like this here overrides `default.env` (but not `.env` itself). Possibly that's what the comment you're deleting from `default.env` was referring to.

That's not necessarily bad, but should probably be noted in `default.env`.